actions/teamscale-upload/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Upload Teamscale Coverage
+
+inputs:
+  partition:
+    description: 'Partition for the test coverage'
+    required: true
+  revision:
+    description: 'Git sha'
+    required: true
+  files:
+    description: 'Path to coverage xml file from jacoco'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+  - name: Upload coverage to teamscale
+    # temporary until we validate that this is working correctly
+    continue-on-error: true
+    uses: 'cqse/teamscale-upload-action@8d10b6693b9242420ef2062dbefc36af6e88d587'
+      with:
+        server: 'https://fdb.teamscale.io'
+        project: 'foundationdb-fdb-record-layer'
+        user: 'fdb-record-layer-build'
+        partition: inputs.partition
+        accesskey: ${{ secrets.TEAMSCALE_ACCESS_KEY }}
```

Comment:
I believe that composite actions can't access secrets directly: https://stackoverflow.com/questions/70098241/using-secrets-in-composite-actions-github

I think the teamscale access key needs to be an input to the step, and then the callers can set the secret. (There are protections in place to stop the secret value from being logged, even if it's provided as input into the action.)